test/api/auth_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class AuthTest < MiniTest::Test
+  include Rack::Test::Methods
+
+  def app
+    Rails.application
+  end
+
+  def test_auth_post
+    # Get response back for logging in with username 'acain' password 'password'
+    post '/api/auth.json', '{"username":"acain", "password":"password"}', "CONTENT_TYPE" => 'application/json'
```

Comment:
There was one little blog post but most of it was Reuben figuring it out :)

We are writing up good documentation for it.
On 9 Apr 2016 4:43 PM, "Alex Cummaudo" <notifications@github.com> wrote:

> In test/api/auth_test.rb
> <https://github.com/doubtfire-lms/doubtfire-api/pull/10#discussion_r59111362>
> :
>
> > @@ -0,0 +1,36 @@
> > +require 'test_helper'
> > +
> > +class AuthTest < MiniTest::Test
> > +  include Rack::Test::Methods
> > +
> > +  def app
> > +    Rails.application
> > +  end
> > +
> > +  def test_auth_post
> > +    # Get response back for logging in with username 'acain' password 'password'
> > +    post '/api/auth.json', '{"username":"acain", "password":"password"}', "CONTENT_TYPE" => 'application/json'
>
> @Reubsinit <https://github.com/Reubsinit> Awesome! Where did you end up
> finding how to do this? Or was it literally just playing with the library
> code?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/doubtfire-lms/doubtfire-api/pull/10/files/4610bdec39a9177c864473ae43762cc47ced3c54#r59111362>
>